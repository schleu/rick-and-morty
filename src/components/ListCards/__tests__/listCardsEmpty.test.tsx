import { render } from '@testing-library/react';
import { ListCard } from '../Xadwa_index';

import '@testing-library/jest-dom'

describe('<ListCard />', () => {

    it('screenshot empty',()=>{
        const {container} = render(<ListCard characters={[]} />);
        expect(container).toMatchSnapshot()
    })

    it('renders a message when isLoading is false and no characters are provided', () => {
        const { getByRole } = render(<ListCard characters={[]} />);
        const message = getByRole('heading',{
            name: 'No characters found'
        });
        expect(message).toBeInTheDocument();
    });

});
