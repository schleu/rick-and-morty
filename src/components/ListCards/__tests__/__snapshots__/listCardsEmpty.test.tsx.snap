// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListCard /> screenshot empty 1`] = `
<div>
  <div
    class="description"
  >
    <h2>
      No characters found
    </h2>
  </div>
</div>
`;
